website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
+    padding-top: 20px;
+    padding-right: 15px;
+}
```

Comment:
> Do we process CSS into `public/` at all? I'm not seeing as much in development.

We've got a bunch of files in `public/js` even in development. I'd expect the same for CSS if we were processing it at all.